package.json

```diff
@@ -8,16 +8,18 @@
     "lib"
   ],
   "scripts": {
-    "prepublish": "npm run eslint && npm test && npm run clean && npm run build && npm run build-examples",
+    "prepublish": "npm run eslint && npm test && npm run clean && npm run build && npm run styleguide:build && cp -a styleguide/* docs/",
```

Comment:
we can consider to create a symbolic link for docs then we don't need to cp & rm.